pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
```

Comment:
Yup, succesfully tested via
* hpi:run (jenkins 1.642.3, from a pipeline and "Provision via OpenStack plugin" dropdown)
* standalone Jenkins (2.47), from the same two methods.

The sample pipeline script is 
`def server = openstack.server ("test","RHEL7")
server.boot()
println "Node floating address ${server.address}"
server.destroy()`

Another anoying element is that the global var doesnt seem to be callable with named parameters, so the invocation has to be as noted